apps/dotcom/src/__snapshots__/routes.test.tsx.snap

```diff
@@ -26,6 +22,10 @@ exports[`the_routes 1`] = `
     "reactRouterPattern": "/r/:roomId",
     "vercelRouterPattern": "^/r/[^/]*/?$",
   },
+  {
+    "reactRouterPattern": "/ro/:roomId",
```

Comment:
We probably need to add a test for this, similarly to how we removed it for `/new`